LPS-135870 WIP

mateussgn opened the pull request:
Related issue: https://issues.liferay.com/browse/LPS-135870

victorg1991 committed:
LPS-135483 Not needed cloneElement

victorg1991 committed:
LPS-135483 Use react memo in collection component

victorg1991 committed:
LPS-135483 Only render itemActions if the element is active

Commit:
LPS-135870 Make Form Field required when Object Field is required

interaminense committed:
LPS-135870 Pass objectField through serializedFormContext

Commit:
LPS-135870 Make sure expressions have access to objectFields

rodrigopaulino committed:
LPS-124588 This logic doesn't make sense. Forms stopped using autosave to persist published versions of form instances ever since LPS-90047. They should always be draft.

rodrigopaulino committed:
LPS-124588 buildLang

Commit:
LPS-137978 Add FragmentsCanBeHiddenAndShown stub

Commit:
LPS-137978 Add widgets and fragments to page

Commit:
LPS-137978 Create macros to hide fragments

Commit:
LPS-137978 Create macros to unhide fragments

Commit:
LPS-137978 Add widgets and fragments to the page

Commit:
LPS-137978 Complete FragmentsCanBeHiddenAndShown test

Commit:
LPS-137978 Add acceptance flag to FragmentsCanBeHiddenAndShown

vicnate5 committed:
LPS-135504 Update test 'does-not-contain'

kevhlee committed:
LPS-138029 Add information on DB tables to upgrade report

Commit:
LPS-137978 Add assert for not present fragments and fix other asserts

Commit:
LPS-137978 Fix LPS in comments

Commit:
LPS-137978 Add HiddenPropertyIsPropagatedToSmallerViewpoints stub

Commit:
LPS-137978 Fix some typos in task names for FragmentsCanBeHiddenAndShown test

Commit:
LPS-137978 Update clickHideFragmentViaPageStructure macro to accept index

Commit:
LPS-137978 Implement case HiddenPropertyIsPropagatedToSmallerViewpoints

Commit:
LPS-137978 Update clickUnhideFragmentViaPageStructure and clickHideFragmentViaPageStructure

Commit:
LPS-137978 Add clickRestoreDefaultHidePropertyForFragment

Commit:
LPS-137978 Add FragmentViewOptionsCanBeResetToParentOne test case

vicnate5 committed:
LPS-135504 Update test 'is-empty'

kevhlee committed:
LPS-138029 SF split code into multiple methods

kevhlee committed:
LPS-138029 SF call Collections.sort() within _getTableInfos()

rodrigopaulino committed:
LPS-124588 Start to set settings in form versions and using them when not in preview mode

rodrigopaulino committed:
LPS-124588 Adapt tests

achaparro committed:
LPS-138029 Use portal-kernel utilities and SF

Commit:
LPS-137978 Fix issues in FragmentViewOptionsCanBeResetToParentOne

kevhlee committed:
LPS-138029 Catch SQLException and return null if it's thrown

kevhlee committed:
LPS-138029 Don't need this anymore

lfbesada committed:
LPS-138258 Create TemplateTransformer to handle template render.

lfbesada committed:
LPS-138258 Copy from portal-impl/src/com/liferay/portlet/portletdisplaytemplate/dependencies/portlet_display_template_error.ftl, no logic changes.

lfbesada committed:
LPS-138258 Copy from modules/apps/portlet-display-template/portlet-display-template-impl/src/main/java/com/liferay/portlet/display/template/internal/PortletDisplayTemplateImpl.java, no logic changes.

lfbesada committed:
LPS-138258 Implement template render with simplest contextObjects

lfbesada committed:
LPS-138258 Add InfoItemFieldValues to contextObjects used to render the template.

lfbesada committed:
LPS-138258 Add an empty InfoItemFieldValues to TemplateInfoItemFieldReader

kevhlee committed:
LPS-138029 Add date to the upgrade report

kevhlee committed:
LPS-138029 Display number of table rows before and after upgrade

kevhlee committed:
LPS-138029 Simply table information by just keeping track of number of rows

robertoDiaz committed:
LPS-137405 Move deletion logic to a scheduled task to avoid race conditions

robertoDiaz committed:
LPS-137405 SF reduce the number of Message Listeners

robertoDiaz committed:
LPS-137405 Add and update current tests

robertoDiaz committed:
LPS-137405 SF (not used)

lfbesada committed:
LPS-138258 Adequate date template to image InfoItemFieldValue

lfbesada committed:
LPS-138258 Adequate image template to image InfoItemFieldValue

robertoDiaz committed:
LPS-137405 Add lang key

Commit:
LPS-137978 Move select frame relative top to tearDown to make it more stable

lfbesada committed:
LPS-138258 Add method to get InfoFieldValue to the provider interface

lfbesada committed:
LPS-138258 Add a LocalizedInfoItemFieldReader implementation to handle InfoField logic

lfbesada committed:
LPS-138258 Use TemplateLocalizedInfoItemFieldReader to build InfoFieldSet.

lfbesada committed:
LPS-138258 Use TemplateLocalizedInfoItemFieldReader to build InfoFieldValue.

lfbesada committed:
LPS-138258 Use proper error template.

Commit:
LPS-138030 Ensure there is a new line between each upgrade process being listed

Commit:
LPS-138030 List modules with pending upgrades in the upgrade report

Commit:
LPS-138030 Add test case for moduleUpgrades

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

marcelabc committed:
LPS-138296 Add method to retrieve the value because can be an array of boolean

marcelabc committed:
LPS-138296 Fix tests

marcelabc committed:
LPS-138296 Add a hidden input to collect the field's value

Preston-Crary committed:
LPS-135504 Rename conflicting key 'untranslated'.

marcelabc committed:
LPS-138296 Make the icon check green in form entries

Preston-Crary committed:
LPS-135504 Remove duplicate key 'not-available'. For portal-impl we have 'N/A' for en but same translation for several of the other languages.

Preston-Crary committed:
LPS-135504 Remove duplicate key 'workflow-is-in-use.-remove-its-assignment-to-x-x-and-x-more-x'.

Preston-Crary committed:
LPS-135504 Rename conflicting key 'text-area'.

Preston-Crary committed:
LPS-135504 Rename 'action.ADD_STRUCTURE' to 'add-metadata-set' and hack it in place.

The hack is harmless and correctly fixes LPS-80363 so that the other action.ADD_STRUCTURE still says what it's supposed to in roles admin. A design type fix would be quite involved and require API and DTD changes just to make our shared action name hack more official.

Preston-Crary committed:
LPS-135504 Revert the ones that were partially migrated.

Commit:
LPS-137978 Fix FragmentsCanBeHiddenAndShown to check preview correctly

Commit:
LPS-137978 Fix HiddenPropertyIsPropagatedToSmallerViewpoints

Preston-Crary committed:
LPS-135504 Fix task clearing identical translations for conflicting keys when the root language does not match.

Commit:
LRAC-9062 Add CanEditButtonToManageUserPermissions testcase

Commit:
LRAC-9062 create deletePendingAllUsers macro

robertoDiaz committed:
LPS-137405 Remove methods that checked if the elements were deleted inline.

Commit:
LPS-138585 Filter groups by VIEW action

Preston-Crary committed:
LPS-135504 Match pattern of other keys.

Preston-Crary committed:
LPS-135504 Update test.

Preston-Crary committed:
LPS-135504 Sort.

javalosjr96 committed:
LPS-138676 Fixed logic in isRootFolder method and added title to Docbar for Documents and Media

erickmonteiroo committed:
LPS-138612 (account-admin-web): Add back button and account name for consistency

erickmonteiroo committed:
LPS-138614 (account-admin-web): Fix Account Groups tab pagination

Commit:
LRQA-69807 Investigate CI failure of Forms#AddFormToUserSiteAndViewNoDuplicates Test Fix

Commit:
LPS-137978 Cleat macros code

markgulacsy committed:
LPS-138720 Escape facet term

Commit:
LPS-137978 Add new moveElementInsideAnExistingContainer macro

Commit:
LPS-137978 Adapt addElementToContainer to be able more than one element into a container

Commit:
LPS-137978 Add GroupsOfFragmentsCanBeHiddenAndShown case

Commit:
LPS-137978 Add steps to GroupsOfFragmentsCanBeHiddenAndShown to test case 7

AliciaGarciaGarcia committed:
LPS-135850 update the parent id of the custom meta tag no to the id of the parent of the property, upgrade it to the id of the property itself

Commit:
LPS-137978 Add sHiddenFragmentsInPagesTemplatesAppearHiddenInPages

Commit:
LRAC-9081 Add UserOnlySeeInvitedPropertiesOrOwnedProperties testcase

Commit:
LRAC-9062 Add VIEW_USER_PERMISSION VIEW_STATUS_PENDING path

Commit:
LRAC-9085 Add AssertNewAdminPermissions testcase

Commit:
LRAC-9055 Fix ViewAllUserList testcase

Commit:
LRAC-9062 Fix CanEditButtonToManageUserPermissions testcase

Commit:
LRAC-9086 Add AssertNewMemberPermissions testcase

Commit:
LRAC-9087 Add SelectAndChangeAnAdminAndMembertoMembers testcase

Commit:
LRAC-9088 Fix SelectandChangeAnAdminAndMemberToAdmins testcase

Commit:
LRAC-9092 Fix OwnerCanChangeUserPermissions testcase

lfbesada committed:
LPS-138481 Fix flaky tests: testOrderByCreateDate

claraizquierdo committed:
LPS-131314 Fix category selection modal by removing cadmin class from iframe body

Hanlf committed:
LRAC-9025 Add tests about Known Individuals Search

achaparro committed:
LPS-138030 No need to create a specific method

achaparro committed:
LPS-138030 Do not hardcode the final schema version

achaparro committed:
LPS-138029 Print the error in the report

jorgediaz-lr committed:
LPS-138734 Remove layoutSetBranchId from ExportImportConfigurations when disabling page versioning

achaparro committed:
LPS-138029 Use the common pattern to get the tables

achaparro committed:
LPS-138029 Use final

achaparro committed:
LPS-138029 Prevent a nullPointer exception

achaparro committed:
LPS-138029 Use a constant

achaparro committed:
LPS-138029 Rename

davidtellolr committed:
LPS-138778 Fix regression bug produced by LPS-94001 in the rename Display Page Template action.

achaparro committed:
LPS-138030 We should not wait for the ReleaseManagerOSGiCommands service, otherwise the component will be initialized later and the report won't be able to take proper initial values in the constructor

achaparro committed:
LPS-138029 No need for a new line here

achaparro committed:
LPS-138029 SF

Commit:
LPS-135150 Take into account the history router base path for the canonical URLs

Preston-Crary committed:
LPS-135504 Remove duplicates with different capitalization.

Preston-Crary committed:
LPS-135504 Make capitalization consistent.

Preston-Crary committed:
LPS-135504 Update english to avoid clearning the translations.

Preston-Crary committed:
LPS-135504 gw -b releng.gradle syncPortalLang and gw buildLang.

Hanlf committed:
LRAC-9025 Update test to assert user after changing pagination

ruben-pulido committed:
LPS-133856 Add portletPreferences when adding a specific porletItem to a Content Page

liuxingye committed:
LPS-138875 Update macro for clicking duplicate question

liuxingye committed:
LPS-138875 Enable anonymous read via system settings

4lejandrito committed:
LPS-138264 Adding cssClass prop to add color to the icons

claraizquierdo committed:
LPS-138264 Add icon class to TreeviewCard component

claraizquierdo committed:
LPS-138264 Add tests for Treeview new iconCssClass prop for nodes

Commit:
LPS-137978 Use new openPreviewMode macro

koor4356 committed:
LPS-135504 Update tests "Is equal to, Is not empty, & Belongs to"

AliciaGarciaGarcia committed:
LPS-135850 remove the try to keep the standard and buy some space

ealonso committed:
LPS-138258 Simplify

ealonso committed:
LPS-138258 The language for information templates is always ftl

ealonso committed:
LPS-138258 Removes unnecessary variables

sandrodw3 committed:
LPS-135483 Remove editables from store and save them locally

sandrodw3 committed:
LPS-135483 Move FragmentContentInteractionsFilter and FragmentContentProcessor inside FragmentContent so we can pass editables as prop

achaparro committed:
LPS-138029 Add the suffix Info for methods used for the output. I kept the suffix also for _getReleaseInfo because it is used indirectaly to generate the output

claraizquierdo committed:
LPS-138908 Fix TreeFilter when all nodes are deselected

riccardo-alberti committed:
COMMERCE-7185 Localize key

riccardo-alberti committed:
COMMERCE-7186 Remove cancel from header buttons

riccardo-alberti committed:
COMMERCE-7187 Fix jsp folder name reference

riccardo-alberti committed:
COMMERCE-7184 displayOrder should be called order in the front

riccardo-alberti committed:
COMMERCE-7183 Use permission aware search instead of portlet permissions

riccardo-alberti committed:
COMMERCE-7188 Use permission aware query ad orderBy

jkappler committed:
LPS-138794 Use redirect URL fallback

victorg1991 committed:
LPS-127677 Disable submit buttons before submitting the form, an reenable then after

AliciaGarciaGarcia committed:
LPS-138401 No longer needed because modals now escape characters (Liferay.Util.openModal is now on react)

sandrodw3 committed:
LPS-135483 Allow adding data attributes to UnsafeHTML and add fragmentEntryLinkId so we can query fragments

sandrodw3 committed:
LPS-135483 Query editables instead of taking them from the store

sandrodw3 committed:
LPS-135483 Build editable manually instead of taking it from store when selecting panels

Commit:
LPS-137978 Add test FragmentsHiddenInMasterPagesAreHiddenInPagesThatUseIt

Commit:
LPS-137978 SF

koor4356 committed:
LPS-135504 Update test "Is not equal to and equal to"

Commit:
LRAC-9089 Add ChangeMultipleUserPermissions testcase

riccardo-alberti committed:
COMMERCE-7188 Autogenerated code (gw buildService)

marco-leo committed:
LPS-135400 Add OneToMany Query for Entries

marco-leo committed:
LPS-135400 BuildService

drewbrokke committed:
LPS-138871 account-admin-web: actions.js - empty arrays are valid input, so just check to see if it is an array

Commit:
LRQA-69808 Forms#AddFormInGlobal Test Fix

Commit:
LPS-138926 Create test stubs for tests related to LPS-135549,LPS-135635,LPS-135389

anabuchmann committed:
LRQA-69883 - Quarantine MyWorkflowTasks tests

Commit:
LPS-138926 SF

vicnate5 committed:
COMMERCE-7127 Second license check no longer needed

Commit:
LRAC-9107 Add CanCreatePropertyAndCancel Test

miltonmc committed:
LPS-138935 Refactor PagesVisitor.mapFields()

Commit:
LRAC-9202 Create test stub

Commit:
LRAC-9054 Upgrade launchAC macro

Commit:
LRAC-9054 Create clickUsersByEmail macro

Commit:
LRAC=9084 Add MemberCanBeSelected testcase

rafaellajordao committed:
LRAC-8586 Paths used by SortSegmentAddMembersModal test

rafaellajordao committed:
LRAC-8586 Add SortSegmentAddMembersModal Test

Commit:
LRAC-9081 Create deleteUserFromProperty macro

Commit:
LRAC-9081 Create assertPropertyNotPresent macro

Commit:
LPS-138938 Fix Objects tests names/testcase

Commit:
LRAC-9081 Update UserLostAccess testcase

Commit:
LRAC-9081 Update UserShouldOnlySeePropertiesAreInvitedByACAdminOrPropertiesWhichUserOwner testcase

Commit:
LRAC-9081 Add UserOnlySeeInvitedPropertiesOrOwnedProperties testcase

Commit:
LRAC-9214 Add test stubs

Commit:
LRAC-9227 Create test stubs

Commit:
LPS-138926 Crete stub for CreateObject#CannotCreateDuplicatedObjectName test

Commit:
LRAC-9095 Add AdminCanChangeUserPermissions testcase

Commit:
LRAC-9082 Update UserLostAccessToProperty testcase

rafaellajordao committed:
LRAC-8488 Add ViewMembershipCardShowsAnonymousUsers Test

marco-leo committed:
LPS-135400 FIX bug in query logic

marco-leo committed:
LPS-135400 Fix Logic

brianchandotcom committed:
COMMERCE-7187 regen

Commit:
LRAC-9082 Update UserLostAccessToProperty testcase

Commit:
LRAC-9095 Add AdminCanChangeUserPermissions testcase

Commit:
LPS-138946 - Create test stubs related to Object Portlet tests

Commit:
LPS-138946 SF

dsitu committed:
COMMERCE-7172 Add commerce destination names to portal destination names

dsitu committed:
COMMERCE-7172 Use portal DestinationNames

brianchandotcom committed:
LPS-135400 as used

brianchandotcom committed:
LPS-138029 SF

Commit:
LRAC-8497 Update viewSegmentCriteriaIndividualAttributes macro

Commit:
LRAC-8497 Add SegmentCriteriasNestCorrectlyInTheCriteriaCard testcase

dsitu committed:
COMMERCE-7172 Remove CommerceDestinationNames

dsitu committed:
COMMERCE-7172 Remove unused CommerceStockQuantityMessageListener

brianchandotcom committed:
LPS-135400 rename

brianchandotcom committed:
LPS-135400 reorder

brianchandotcom committed:
LPS-135400 SF / wordsmith

liulukiwi committed:
LRQA-69877 Add property to deploy Google Drive Shortcut

liulukiwi committed:
LPS-138907 Modify case name and portal version to dump version

Hanlf committed:
LPS-138887 Update the option's index

Hanlf committed:
LPS-138872 Use URL to open blog entry

Commit:
LPS-138896 Migrate cases

liulukiwi committed:
LPS-138906 Add condition for 7.0.6 version bundle

Commit:
LPS-138896 Add case to view collection items based on Basic Web Content on display page template for Category

Hanlf committed:
LPS-135222 Assert no pop up should be enough

liulukiwi committed:
LRQA-69877 Add property to deploy Google Drive Shortcut

Commit:
LPS-138896 Add case to view collection items based on Blogs on display page template for Category

Commit:
LPS-138896 Modify locator to make it reusable

Commit:
LPS-138896 Add case to view collection items based on Basic Document on display page template for Category

lfbesada committed:
LPS-138911 Fix JournalArticleServiceTest.testCheckArticleWithInvalidStructure.

The fix for LPS-133617 avoids caching journal content, which causes the default values to be sets in document if content doesn't have some structure field.

We change put the desired document to test the specific case.

jkappler committed:
LPS-138794 Not used

austinchiang committed:
LPS-136192 Change setup order

Commit:
LPS-138896 Add case to view collection items based on custom web content structure on display page template for Category

jkappler committed:
LPS-138794 Extract to display context

Commit:
LPS-138896 Add case to view collection items based on custom document type on display page template for Category

jkappler committed:
LPS-138794 SF so that it matches dynamic view

Commit:
LPS-138896 Add case to preview mapped content of collection item on display page template for Category

jkappler committed:
LPS-138971 Unregister service registrations when components are deactivated

andrea-ale-sbarra committed:
COMMERCE-7163 headless-commerce-shop-by-diagram-impl removed unused parameters (yaml file)

andrea-ale-sbarra committed:
COMMERCE-7163 headless-commerce-shop-by-diagram-impl make the attached dataset show only diagram entries of this Diagram

andrea-ale-sbarra committed:
COMMERCE-7163 headless-commerce-shop-by-diagram autogenerated code (gw buildREST)

victorg1991 committed:
LPS-138436 Filter not present collections to work with deleted collections

riccardo-alberti committed:
COMMERCE-7190 Rename keywords to search for consistency in yaml

riccardo-alberti committed:
COMMERCE-7190 Use search instead of keywords

Commit:
LPS-138896 Add case to view available mappable fields of collection item on display page template for Category

riccardo-alberti committed:
COMMERCE-7190 Autogenerated code (gw buildREST)

Commit:
LPS-138896 Sort cases

p2kmgcl committed:
LPS-135483 Remove editablesReducer

p2kmgcl committed:
LPS-135483 Do not query editableElement until needed

Hanlf committed:
LPS-119421 Add sqlserver database

Hanlf committed:
LPS-138888 Upgrade from EE version

Hanlf committed:
LPS-138888 Remove from quarantine list

riccardo-alberti committed:
COMMERCE-7192 Add actions mapping to DTO converter

riccardo-alberti committed:
COMMERCE-7192 Add exception mapper for user email address validation

liferay-translation committed:
LPS-77699 Update Translations

sandrodw3 committed:
LPS-135483 Take document from GlobalContext

brianchandotcom committed:
COMMERCE-7192 wordsmith

brianchandotcom committed:
LPS-137405 regen

brianchandotcom committed:
LPS-137405 Wordsmith

anabuchmann committed:
LRQA-69907 - Quarantine ViewInvitedResourceAvailability test

brianchandotcom committed:
LPS-138734 SF

Commit:
LPS-135870 Adapt DDMForm to receive objectFields

Commit:
LPS-135870 Adapt IsRequiredObjectFieldFunction to use objectFields parameter

Commit:
LPS-135870 Update Required field rules to support Object Fields

Commit:
LPS-135870 baseline
